### PR TITLE
chore(langgraph): Adds test case for Command.PARENT update behavior

### PR DESCRIPTION
**Pull request by jacoblee93**
Nails down behavior that two updates are expected in this case

### CODE REVIEW

File: libs/langgraph/src/tests/pregel.test.ts [hunk 1]
```diff
@@ -9209,6 +9209,134 @@ graph TD;
...
+        name: "get_user_name",
+        schema: z.object({}),
+      }
+    );
+    const subgraph = new StateGraph(CustomStateAnnotation)
+      .addNode("init", async () => {
+        return { user_name: "Woof" };
+      })
+      .addNode("foo", async () => {})
+      .addNode("tool", getUserName, { ends: ["foo", "__end__"] })
```
[the remark this answers is not in the document]
Comment (benjamincburns) [reply libs/langgraph/src/tests/pregel.test.ts:9241]: also it'd be helpful to add a comment that explains what this test is trying to validate - I'm guessing it's something rather specific/esoteric/edge caseish, but I can't discern exactly what it's meant to be from the code alone.

[the remark this answers is not in the document]
Comment (jacoblee93) [reply libs/langgraph/src/tests/pregel.test.ts:9299]: Yeah doesn't look like it was necessary - probably an oversight
